### PR TITLE
data:parveshsaini.json

### PR DESCRIPTION
**Pull request by parveshsaini**

### CODE REVIEW

**Review by lewisushindi — APPROVED** [review]

File: data/parveshsaini [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      "name": "Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/parvesh-saini-a5345a229/",
+      "icon": "FaLinkedin"
+    },
+        {
+      "name": "Checkout my Open-Source contributions ",
+      "url": "https://github.com/parveshsaini",
+      "icon": "FaGithub"
+    }
+  ]
```
Comment (senali-d) [inline data/parveshsaini:23]: `,` is missing
```suggestion
  ],
```

**Review by senali-d — CHANGES_REQUESTED** [review]
Looks good, but do the suggested change.

**Review by dan-mba — CHANGES_REQUESTED** [review]
please add `.json` to the end of your filename.

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, 

Dan means adding `.json` to your file which you added in the data folder. YOu can see in the screenshot the file name without extension.

Screenshot 2023-01-25 at 1 31 25 AM

**Review by devvsakib — APPROVED** [review]
LGTM

**Review by devvsakib — CHANGES_REQUESTED** [review]
change your file name to `parveshsaini.json`

**Pull request closed without merging**